src/style.css

```
@@ -1 +1,46 @@
 /* add styles here */
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  flex-direction: row;
+  align-items: center;
+  list-style-type: none;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center ;
+  background-repeat: no-repeat;
+}
+
+.stars--1 li:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 li:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 li:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--1 li:nth-child(-n + 1),
.stars--2 li:nth-child(-n + 2),
.stars--3 li:nth-child(-n + 3) {
  background-image: url(images/star-active.svg);
}
```
combine it with a comma